src/main/java/org/folio/notes/mapper/TypeMapper.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.notes.mapper;
+
+import java.sql.Timestamp;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.UUID;
+
+import org.apache.commons.lang3.StringUtils;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValueCheckStrategy;
+import org.springframework.data.domain.Page;
+
+import org.folio.notes.domain.dto.Metadata;
+import org.folio.notes.domain.dto.NoteType;
+import org.folio.notes.domain.dto.NoteTypeCollection;
+import org.folio.notes.domain.entity.TypeEntity;
+
+@Mapper(componentModel = "spring", nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS)
+public interface TypeMapper {
+
+  @Mapping(target = "metadata", expression = "java(toMetadata(entity))")
+  NoteType toDto(TypeEntity entity);
+
+  @InheritInverseConfiguration
+  TypeEntity toEntity(NoteType dto);
+
+  default NoteTypeCollection toDtoCollection(Page<TypeEntity> entityList) {
+    return new NoteTypeCollection().noteTypes(toDtoList(entityList.getContent())).totalRecords(
+      Math.toIntExact(entityList.getTotalElements()));
```

Comment:
```suggestion
    return new NoteTypeCollection()
    .noteTypes(toDtoList(entityList.getContent()))
    .totalRecords(Math.toIntExact(entityList.getTotalElements()));
```
Could you please unwrap it on dots? It will be more readable.